storage/healthcheck.go

```diff
@@ -16,24 +16,33 @@
 
 package storage
 
-import "github.com/Peripli/service-manager/pkg/health"
+import (
+	"github.com/InVisionApp/go-health/checkers"
+	"github.com/Peripli/service-manager/pkg/health"
+)
+
+func NewHealthIndicator(pingFunc PingFunc) (health.Indicator, error) {
+	sqlConfig := &checkers.SQLConfig{
+		Pinger: pingFunc,
+	}
+	sqlChecker, err := checkers.NewSQL(sqlConfig)
```

Comment:
this is implementation specific in the storage package. Move this to postgres package or rename this to `SQLHealthIndicator` / `PostgresHealthIndicator`.